shim/internal/config_test.go

```diff
@@ -13,28 +14,75 @@ import (
 	"time"
 
 	. "github.com/hyperledger/fabric-chaincode-go/shim/internal"
+	peerpb "github.com/hyperledger/fabric-protos-go/peer"
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/keepalive"
 )
 
-var keyPEM = `-----BEGIN EC PRIVATE KEY-----
-MHcCAQEEIMLemLh3+uDzww1pvqP6Xj2Z0Kc6yqf3RxyfTBNwRuuyoAoGCCqGSM49
-AwEHoUQDQgAEDB3l94vM7EqKr2L/vhqU5IsEub0rviqCAaWGiVAPp3orb/LJqFLS
-yo/k60rhUiir6iD4S4pb5TEb2ouWylQI3A==
------END EC PRIVATE KEY-----
+var keyPEM = `-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgKg8jpiNIB5LXLull
+IRoYMsQximSiU7XvGCYLslx4GauhRANCAARBGdslxalpg0dxk9GwVhi+Qw9oKZPE
+n1hWPFmusDKtNbDLsHd9k1lU+SWnJKYlg7hmaUvxC1lR2M6KmvAwSUfN
+-----END PRIVATE KEY-----
```

Comment:
Can we get a comment about how the keys and certs were generated? Eventually someone will want to know how this was.

Reply:
Fair enough. Added a comment and pushed (in this case simply used cryptogen to generate the artifacts).

Reply:
This isn't holding up a merge but specifics are generally better. Saying we used cryptogen is like saying we used openssl; it's not enough for someone to recreate.